decidim/decidim
# Do not show scopes column in budgets if there isn't subscopes

#### :tophat: What? Why?
Scopes within Budgets were not editable and remained within a global scope. This PR allows this change to take place. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #11390 
- Fixes #12018 

#### Testing
1. Login as an admin
2. Head over to an module like Voting
3. Make sure in the budget components you have "Scopes enabled"
4. Head back and click on a budget
5. Edit that budget
6. See that scopes are now enabled 

### :camera: Screenshots
Scopes editable: 
![Screenshot 2023-11-16 at 16 13 48](https://github.com/decidim/decidim/assets/101816158/cf69d85c-a282-4d7c-a32d-e5a39ec3b824)

Index: 
![Screenshot 2023-11-16 at 16 14 55](https://github.com/decidim/decidim/assets/101816158/1d5b8bdb-6d59-4932-808a-5d0a12a76334)


:hearts: Thank you!
